app/components/Article/index.js

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
-import { addHighlight } from 'actions/article';
+import { addHighlight, deleteHighlight, selectHighlight } from 'actions/article';
```

Comment:
Sounds good - initially all the actions were placed in the action folder. In the future I'll put all the "module" files in a separate directory.